charts/sonarqube-dce/templates/config.yaml

```diff
@@ -32,15 +29,12 @@ metadata:
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
 data:
-  {{- if and .Values.sonarSecretKey (not .Values.searchNodes.sonarProperties) }}
-  sonar.properties: sonar.secretKeyPath={{ .Values.sonarqubeFolder }}/secret/sonar-secret.txt
-  {{- end }}
-  {{- if or .Values.searchNodes.sonarProperties }}
+{{- if or .Values.sonarSecretKey .Values.searchNodes.sonarProperties }}
```

Comment:
similar comment as in the standard chart. Superfluous condition that might stay for readability purposes.